tests/flow_validation/test_action_state.py

```diff
@@ -0,0 +1,326 @@
+import typing as t
+
+import pytest
+
+from globus_automate_client.models import FlowDefinition, FlowValidationError
+
+simple_action_state = {
```

Comment:
Looks like all of these are single step flows, so we probably could just test directly on the state of interest rather than providing a full flow definition (though we'd need separate test cases for the fields in the flow definition and how that then interacts with unreferenced state names, etc.)

Reply:
This is really a great idea - it removes complexity/indentation in the test cases and removes a potential source for exceptions that get raised.